### PR TITLE
*: add Makefile and make check happy.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
PTAL

### CODE REVIEW

File: coprocessor.go [hunk 1]
```diff
@@ -27,10 +27,12 @@ import (
 	"github.com/pingcap/tipb/go-tipb"
 )
 
+// CopClient is coprocessor client.
 type CopClient struct {
 	store *tikvStore
 }
 
+// SupportRequestType checks whether reqType is supported now.
```
Comment (ngaut) [inline coprocessor.go]: Remove "now"

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	go get github.com/golang/lint/golint
+
+	go tool vet .
+	go tool vet --shadow .
+	golint ./...
+	gofmt -s -l .
+
+deps:
+	go list -f '{{range .Deps}}{{printf "%s\n" .}}{{end}}{{range .TestImports}}{{printf "%s\n" .}}{{end}}' ./... | \
+		sort | uniq | grep -E '[^/]+\.[^/]+/' |grep -v "pingcap/pd" | \
```
Comment (qiuyesuifeng) [inline Makefile]: Rename `pd` to `ticlient`.

File: oracle/oracle.go [hunk 3]
```diff
@@ -13,6 +13,7 @@
 
 package oracle
 
+// Oracle is the timestamp manager interface.
```
Comment (qiuyesuifeng) [inline oracle/oracle.go]: @disksing  What is your option of this comment?

Comment (disksing) [inline oracle/oracle.go]: How about `Oracle is the interface that provides strictly ascending timestamps`?

Comment (qiuyesuifeng) [inline oracle/oracle.go]: Good suggestion.

**Comment by disksing** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**